test/fixtures/output/kafka/mirror/truncate-and-restore-mirror-json.golden

```diff
@@ -6,7 +6,7 @@
     "last_source_fetch_offset": -1,
     "error_message": "Not authorized",
     "error_code": "401",
-    "messages_truncated": 0,
+    "messages_truncated": "0",
```

Comment:
Do we have a test where `messages_truncated` is `""`? (Which is the reason why we're using a `string` here instead of an `int64`)